.gitignore

```diff
@@ -0,0 +1,2 @@
+/.vscode/
```

Comment:
please remove .vscode it is not related to the package

Reply:
Removed